components/match2/commons/match_group_display_helper.lua

```diff
@@ -108,15 +109,15 @@ end
 -- Display component showing the streams, date, and countdown of a match.
 function DisplayHelper.MatchCountdownBlock(match)
 	DisplayUtil.assertPropTypes(match, MatchGroupUtil.types.Match.struct)
-	local dateFormatString
+	local dateString
 	if match.dateIsExact == true then
-		dateFormatString = 'F j, Y - H:i'
+		dateString = mw.getContentLanguage():formatDate('F j, Y - H:i', match.date) .. ' ' .. _UTC
```

Comment:
```suggestion
		dateString = mw.getContentLanguage():formatDate('F j, Y - H:i', match.date) .. _UTC
```the space is not needed it gets trimmed away in the countdown module anyways
and without it the code looks cleaner imo